site/src/github.rs

```diff
@@ -566,23 +570,61 @@ async fn post_comparison_comment(ctxt: &SiteCtxt, commit: QueuedCommit, is_maste
     post_comment(&ctxt.config, pr, body).await;
 }
 
+fn make_comparison_url(commit: &QueuedCommit, stat: Stat) -> String {
+    format!(
+        "https://perf.rust-lang.org/compare.html?start={}&end={}&stat={}",
+        commit.parent_sha,
+        commit.sha,
+        stat.as_str()
+    )
+}
+
 async fn summarize_run(ctxt: &SiteCtxt, commit: QueuedCommit, is_master_commit: bool) -> String {
-    let comparison_url = format!(
-        "https://perf.rust-lang.org/compare.html?start={}&end={}",
-        commit.parent_sha, commit.sha
+    let benchmark_map = ctxt.get_benchmark_category_map().await;
+
+    let mut message = format!(
+        "Finished benchmarking commit ({sha}): [comparison url]({comparison_url}).
+
+**Summary**: ",
+        sha = commit.sha,
+        comparison_url = make_comparison_url(&commit, Stat::Instructions)
```

Comment:
Does it make sense to link here now?

Reply:
Well, maybe it's not strictly needed, but I think that people are used to it by now and having a visible link at the beginning of the summary seems like a useful thing. But I can remove it if you want.